src/Aura/Http/Adapter/Stream.php

```diff
@@ -314,6 +314,11 @@ protected function setContext()
 
         // what scheme are we using?
         $url = parse_url($this->request->url);
+
+        if (empty($url['scheme'])) {
+            $url['scheme'] = 'http';
```

Comment:
I have a doubt : Doesn't this means, the scheme is always http when it is empty?